### DISCUSSION TITLE
Text normalization and inverse text normalization

### DISCUSSION THREAD

**Discussion by desh2608** [lhotse-speech/lhotse#825] [Ideas]
Some of the recipes already provide basic Kaldi-style text normalization, but it might be useful to add dedicated workflows for this. In the style of `annotate_with_whisper`, we could have a workflow that does `normalize_with_nemo` using NeMo's WFST based [text normalization tool](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/v1.0.0/nemo_text_processing/text_normalization.html).

**Comment by pzelasko** [comment]
great idea!
